src/views/adminDashboard.js

```diff
@@ -16,46 +17,14 @@ function AdminDashboard() {
 
 
     return (
-<>
+        <>       
             <AdminNavbar />
-            <Container>
-                <Button className="mt-4 mb-4" variant="primary" onClick={handleShowCreateParkLotModal} data-test-id='create-park-lot-btn'>
-                    Create Parking Lot
```

Comment:
Let's keep these Create/Delete parking lot buttons for now and deal with them in a separate PR